server/Mistakes.Journal.Api/Logic/Mistakes/Models/Mistake.cs

```diff
@@ -38,5 +41,10 @@ public enum MistakePriority
             Medium = 2,
             High = 4,
         }
+
+        public bool CanBeSolved()
+        {
+            return (DateTime.Now - CreatedAt).TotalDays >= Constants.DaysToSolveMistake;
+        }
```

Comment:
There is a good practice, to don't add methods on EF Core classes. I would make it as a extension for mistake :)

Reply:
so stackoverflow is lying xD